BrowserKit/Package.swift

```diff
@@ -29,13 +29,15 @@ let package = Package(
     targets: [
         .target(
             name: "SiteImageView",
-            dependencies: ["Fuzi", "Kingfisher", "Common"]),
+            dependencies: ["Fuzi", "Kingfisher", "Common"],
+            swiftSettings: [.unsafeFlags(["-enable-testing"])]),
```

Comment:
I had to add this so that the Unit Tests for Browserkit run successfully. Otherwise we were having compilation errors: `Module Common was not compiled for testing`

I followed the solution given in [this article](https://antran.app/2021/custom_build_configuration_swift_package_tests/) that I found while searching for the error

Reply:
@OrlaM we discussed this over Slack and thought it'd be okay for now (until SPM better supports our needs). What are your thoughts on this?

Context, the linked article & https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#swiftsetting

Reply:
Not ideal but it will do as a work around for now. Good find 👍